modules/sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service.adoc

```diff
@@ -41,82 +37,61 @@ EOD
 The output verifies that the `ServiceBinding` CR is created to project the binding data into the sample application.
 +
 .Example output
+[source,terminal]
 ----
-servicebinding.binding.operators.coreos.com/spring-petclinic-rest created
+servicebinding.binding.operators.coreos.com/spring-petclinic created
 ----
 
-. To verify that the binding is successful, check the status conditions of the binding resource:
+. Verify that the request for service binding is successful:
 +
+[source,terminal]
 ----
-$ oc get servicebindings spring-petclinic-rest -n my-postgresql -o jsonpath-as-json='{.status.conditions}'
+$ oc get servicebindings -n my-petclinic
 ----
 +
 .Example output
+[source,terminal]
 ----
-[
-  [
-    {
-      "lastTransitionTime": "2021-09-06T13:42:28Z",
-      "message": "",
-      "reason": "DataCollected",
-      "status": "True",
-      "type": "CollectionReady",
-    },
-    {
-      "lastTransitionTime": "2021-09-06T13:42:28Z",
-      "message": "",
-      "reason": "ApplicationUpdated",
-      "status": "True",
-      "type": "InjectionReady",
-    },
-    {
-      "lastTransitionTime": "2021-09-06T13:42:28Z",
-      "message": "",
-      "reason": "ApplicationsBound",
-      "status": "True",
-      "type": "Ready",
-    },
-  ],
-]
+NAME                          READY   REASON              AGE
+spring-petclinic-postgresql   True    ApplicationsBound   47m
```

Comment:
```suggestion
spring-petclinic-pgcluster   True    ApplicationsBound   47m
```
The name of the `ServiceBinding` resource is `spring-petclinic-pgcluster`

Reply:
Done.